README.md

```diff
@@ -1,10 +1,11 @@
-# winbatchscripts
-Various Batch scripts for Windows.
+# WinBatch Scripts
+Various Batch scripts for Windows.  
+Authored by: Matthew Carras  :pen:
 
-RunAsInvoker.bat - Simple script designed to be put in your "Send To" folder which runs the file as the current user, bypassing UAC checks and running it with whatever privilege level is currently set. Useful if the program always demands Administrator privileges but doesn't actually need them.
+**RunAsInvoker.bat** - Simple script designed to be put in your "Send To" folder which runs the file as the current user, bypassing UAC checks and running it with whatever privilege level is currently set. Useful if the program always demands Administrator privileges but doesn't actually need them.
```

Comment:
Simple Markdown highlighting can enhance the readability of documents on GitHub.

💡  **Possible** consideration ...